testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ScenarioApplication.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package dev.flutter.scenarios;
+
+import android.os.StrictMode;
+import io.flutter.app.FlutterApplication;
+
+public class ScenarioApplication extends FlutterApplication {
```

Comment:
just double checking. Our CI doesn't run this yet so I'm assuming you ran it manually to check.

Reply:
Yes - I've run this manually on Firebase and on local devices.

The LUCI script is attempting to run the scenario app test on Firebase (https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/engine.py#560).  But it looks like this is currently broken on the LUCI bots.